deploy/olm/bundle/0.0.1/starboard-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -1,26 +1,121 @@
 apiVersion: operators.coreos.com/v1alpha1
 kind: ClusterServiceVersion
```

Comment:
You can preview operator's package at https://operatorhub.io/preview

Reply:
This files is just a template. Eventually we'll submit it as PR to https://github.com/operator-framework/community-operators